run_tests.py

```diff
@@ -1,18 +1,59 @@
+import os
+import subprocess
+from tempfile import TemporaryDirectory
+
 # Force COLLECT_LOGS=True
 # Force RISK_MODEL=True
+# Fix RISK_MAPPING_FILE relative path
 import config
 config.COLLECT_LOGS = True
 config.RISK_MODEL = "first order probabilistic tracing"
+config.RISK_MAPPING_FILE = os.path.join(os.path.dirname(__file__), config.RISK_MAPPING_FILE)
+
+
+def start_inference_server(root_dir=None):
+    if root_dir is None:
+        root_dir = os.path.dirname(__file__)
+    working_dir = os.path.join(root_dir, "covid_p2p_risk_prediction")
+    # TODO: use mila-iqia once satyaog's branch gets merged
+    subprocess.run(
+        f"git clone -binference-server --depth=1 "
+        f"https://github.com/satyaog/covid_p2p_risk_prediction.git {working_dir}".split(),
+        stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL
+    )
```

Comment:
using mine until [it gets merge](https://github.com/mila-iqia/covid_p2p_risk_prediction/pull/14) on mila-iqia

Reply:
there's a mini change request in that branch